docs-src/cloud/prometheus-grafana-setup-cloud.md

```diff
@@ -247,7 +252,7 @@ Verify your Prometheus configuration and restart Prometheus.
 If you’re running Grafana as a container, you can set your SDK metrics Prometheus data source in your Grafana configuration.
 See the example Grafana configuration described in the [Prometheus and Grafana setup for open-source Temporal Cluster](/kb/prometheus-grafana-setup) KB article.
 
-## Grafana dashboards setup
+### Grafana dashboards setup
```

Comment:
Correct this: "In this article, we will configure our dashboards using the UI." on line 259 to "In this section, we will configure our dashboards using the UI."